README.md

```diff
@@ -14,16 +14,133 @@ At the moment, it supports transferring data between [file locations](https://gi
+
+   https://github.com/astronomer/apache-airflow-provider-transfers/blob/04b53d780790eaa3b424458742bc89d6fbec2ccd/example_dags/example_transfer_and_return_files.py#L1-L45
+
+   https://github.com/astronomer/apache-airflow-provider-transfers/blob/a80dc84b7f33bb86ae244f79411b240f4f4c7e22/example_dags/example_snowflake_transfers.py#L1-L46
+
+   Alternatively, you can download `example_transfer_and_return_files.py`
+   ```shell
+    curl -O https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_transfer_and_return_files.py
+   ```
+
+   Alternatively, you can download `example_snowflake_transfers.py`
+   ```shell
+   curl -O https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_snowflake_transfers.py
+   ```
+
+  * To start running your project in a local Airflow environment, run the following command from your project directory:
+
+    ```shell
+    astro dev start
+    ```
+  * After your project builds successfully, open the Airflow UI in your web browser at https://localhost:8080/. Find your DAGs in the dags directory in the Airflow UI.
+  * Create airflow connection for snowflake, google, SFTP and amazon using airflow UI and run the DAGs.
+
+
+### Run `UniversalTransferOperator` using [airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/installation/index.html)
+  * Install airflow and setup project following this [documentation](https://airflow.apache.org/docs/apache-airflow/stable/installation/index.html).
+  * Ensure that your Airflow environment is set up correctly by running the following commands:
+
+    ```shell
+    export AIRFLOW_HOME=`pwd`
+    airflow db init
+    ```
+
+  * Add the following in requirements.txt
+
+    ```text
+    apache-airflow-provider-transfers[all]
+    ```
+
+  * Copy file named [example_transfer_and_return_files.py](./example_dags/example_transfer_and_return_files.py) and [example_snowflake_transfers.py](./example_dags/example_snowflake_transfers.py) and add it to the `dags` directory of your Airflow project:
+
+   https://github.com/astronomer/apache-airflow-provider-transfers/blob/04b53d780790eaa3b424458742bc89d6fbec2ccd/example_dags/example_transfer_and_return_files.py#L1-L45
+
+   https://github.com/astronomer/apache-airflow-provider-transfers/blob/a80dc84b7f33bb86ae244f79411b240f4f4c7e22/example_dags/example_snowflake_transfers.py#L1-L46
+
+   Alternatively, you can download `example_transfer_and_return_files.py`
+   ```shell
+    curl -O https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_transfer_and_return_files.py
+   ```
+
+   Alternatively, you can download `example_snowflake_transfers.py`
+   ```shell
+   curl -O https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_snowflake_transfers.py
+   ```
```

Comment:
Alternatively, you can download both `example_transfer_and_return_files.py` and `example_snowflake_transfers.py` using the command line:
   
   ```shell
    curl -O https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_transfer_and_return_files.py
    curl -O https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/example_dags/example_snowflake_transfers.py